tools/gcp/comp_test/create_comptest_images.sh

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
```

Comment:
also, a README.md file would be useful in this directory, explaining the purpose of these tests.